std/zip.d

```diff
@@ -387,6 +399,91 @@ final class ArchiveMember
         printf("\tindex = x%08x\n", index);
     }
     }
+
+    /**
+     * Checks the contents of this archive for typical traces of malware.
```

Comment:
OK. I did not know about that. I'll change this.

Reply:
"unusual and potentially ill-intentioned deviations" is a little bit bulky. What about "potentially harmful data"?

By the way, I meanwhile noticed that I came up with "malware", because the original issue contained that word in the title - I just continued to use it, without actually thinking about it. Having read more about what it means, I think, malware is mainly used for "bad programms" and not for "bad data". So now I'm convinced, that it is a good idea to replace it with something better. ;)